src/IO/InputStreamFromROOTFileForCylindricalPET.cxx

```diff
@@ -146,6 +148,13 @@ InputStreamFromROOTFileForCylindricalPET::set_defaults()
     module_repeater_y = -1;
     module_repeater_z = -1;
     rsector_repeater = -1;
+#if STIR_ROOT_AS_V4
+    half_block = module_repeater_y * submodule_repeater_y * crystal_repeater_y / 2  - 1;
+    if (half_block < 0 )
+      half_block = 0;
+#else
+    half_block = 0;
+#endif
```

Comment:
Keeping `half_block` for now. Can be removed when `STIR_ROOT_AS_V4=false` later.